src/Zigzag_persistence/doc/Intro_zigzag_persistence.h

```diff
@@ -0,0 +1,52 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Hannah Schreiber
+ *
+ *    Copyright (C) 2023 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#ifndef DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+#define DOC_ZIGZAG_PERSISTENCE_INTRO_ZIGZAG_PERSISTENCE_H_
+
+// needs namespace for Doxygen to link on classes
+namespace Gudhi {
+namespace zigzag_persistence {
+
+/** \defgroup zigzag_persistence Zigzag Persistence
+ * @{
+ * \author    Cl&eacute;ment Maria, Hannah Schreiber
+ *
+ * We refer to the introduction page \ref persistent_cohomology for persistent (co)homology for an introduction
+ * to the topic.
+ * Zigzag persistence is a generalization of the latter. While standard persistence only allows to grow the filtered
+ * complex by adding simplices, zigzag persistence also allows removals. Hence the name "zigzag", as the module
+ * diagram will have arrows alterning between forward and backward.
+ *
+ * The implementation is based on the algorithm introduced in \cite zigzag.
+ *
+ * \subsection zigzaginterface Stream-like interface
```

Comment:
The choice stream/store does not look orthogonal to the choice filtered/not, in the classes that are available. Is it because we expect everyone will want the filtered version?

Reply:
You mean, why filtered have both a stream and store version while not-filtered only have a stream one?
I assumed indeed that the filtered one will much more used. But also because the only difference it would make for the not-filtered version is that one will store a vector while the other stores a callback method. The filtered versions are also not very different, but there could be some difference in performance because of the way the filtration values are differently stored, while the not-filtered performance would only depend on the callback method.